lab_3/main.py

```diff
@@ -59,42 +108,79 @@ def encode_corpus(storage: LetterStorage, corpus: tuple) -> tuple:
     :param corpus: a tuple of sentences
     :return: a tuple of the encoded sentences
     """
-    pass
+    if not isinstance (corpus, tuple) or type(storage) in [int, str, list, tuple, dict, float, bool] or \
+            storage is None:
```

Comment:
can be shortened.

Reply:
i don't get how to check the type class

Reply:
the `is None` check is ~equal to "object is nothing", so the `obj is None` can be shortened into `not object`.